motor_joints:
  robot_movement_joints:
    left_front_joint: &movement_left_front_joint left_front_wheel_joint
    right_front_joint: &movement_right_front_joint right_front_wheel_joint
    left_rear_joint: &movement_left_rear_joint left_rear_wheel_joint
    right_rear_joint: &movement_right_rear_joint right_rear_wheel_joint

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

skid_steer_drive_controller:
  type: pandora_motor/SkidSteerDriveController
  wheel_separation: 0.39757
  wheel_radius: 0.088895
  max_angular_velocity: 22.5
  joints:
    - *movement_left_front_joint
    - *movement_right_front_joint
    - *movement_left_rear_joint
    - *movement_right_rear_joint
